app/controllers/api/v2/users_controller.rb

```diff
@@ -111,6 +112,22 @@ def update
         end
       end
 
+      api :PATCH, "/users/invalidate_jwt", N_("Invalidate all JSON Web Tokens (JWTs) for a user or users.")
```

Comment:
This now looks like it invalidates it for all users and there's no way to specify it for users. REST API wise I think this is also poor design. If you have a user that's called `invalidate_jwt` then it overlaps with other resources.

I'd expect `DELETE /users/:id/jwt` to delete all JWTs for a specific user while for all JWTs it should operate on the JWT collection, so something like `DELETE /jwt`. We currently don't have such an endpoint so you'd need to create that.

Reply:
I think a user with `invalidate_jwt` would be an edge case
we are using scoped search approach in the API request so the user can use id or name or login anything for reference. From API POV, its valid for one specific user or multiple users as well. Request params would go like "{"search" : "id ^ (1, 2, 3)" } , does it make sense?

Reply:
cc @ShimShtein